cyclones.R/Task2.R

```diff
@@ -0,0 +1,8 @@
+## Task 2
+
+read_data <- function(url, destfile) {
+  download.file (url = url, destfile = destfile)
+  
+  read_xlsx(file = destfile)
+}
```

Comment:
I will not put this one here. I will instead source the function from the R folder that contains the R function you created.